undertow/src/main/java/org/wildfly/extension/undertow/HostDefinition.java

```diff
@@ -89,12 +89,20 @@
             .setDefaultValue(ModelNode.TRUE)
             .setAllowExpression(true)
             .build();
+    static final SimpleAttributeDefinition ACTIVE_REQUEST_TRACKING_ENABLED = new SimpleAttributeDefinitionBuilder(Constants.ACTIVE_REQUEST_TRACKING_ENABLED, ModelType.BOOLEAN, true)
+            .setRestartAllServices()
+            .setDefaultValue(ModelNode.FALSE)
+            .setAllowExpression(true)
+            .build();
```

Comment:
This new attribute requires a transformer to handle the case where this attribute is defined and the model needs to be transformed for a host using an older version of the model.